cmd/copy.go

```diff
@@ -1070,7 +1070,7 @@ func validateMetadataString(metadata string) error {
 	if err != nil {
 		return err
 	}
-	for k, _ := range metadataMap {
+	for k := range metadataMap {
```

Comment:
How is this a linting issue? Range shall be returning back index and item and this shall not compile if it was wrong earlier.